[32mINFO[39m:     Started server process [[36m7556[39m]
[32mINFO[39m:     Waiting for application startup.
[32mINFO[39m:     Application startup complete.
[32mINFO[39m:     127.0.0.1:61230 - "[1mGET / HTTP/1.1[22m" [32m200 OK
[32mINFO[39m:     127.0.0.1:61230 - "[1mGET /favicon.ico HTTP/1.1[22m" [31m404 Not Found
[32mINFO[39m:     127.0.0.1:61251 - "[1mGET /docs HTTP/1.1[22m" [32m200 OK
[32mINFO[39m:     127.0.0.1:61251 - "[1mGET /openapi.json HTTP/1.1[22m" [32m200 OK
[32mINFO[39m:     127.0.0.1:61317 - "[1mPOST /predict HTTP/1.1[22m" [91m500 Internal Server Error
[31mERROR[39m:    Exception in ASGI application
Traceback (most recent call last):
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\uvicorn\protocols\http\h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\uvicorn\middleware\proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\fastapi\applications.py", line 259, in __call__
    await super().__call__(scope, receive, send)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\starlette\applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\starlette\middleware\errors.py", line 181, in __call__
    raise exc
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\starlette\middleware\errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\starlette\exceptions.py", line 82, in __call__
    raise exc
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\starlette\exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\fastapi\middleware\asyncexitstack.py", line 21, in __call__
    raise e
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\fastapi\middleware\asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\starlette\routing.py", line 656, in __call__
    await route.handle(scope, receive, send)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\starlette\routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\starlette\routing.py", line 61, in app
    response = await func(request)
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\fastapi\routing.py", line 226, in app
    raw_response = await run_endpoint_function(
  File "C:\Users\Tiago Barreto\anaconda3\envs\colab2deploy\lib\site-packages\fastapi\routing.py", line 159, in run_endpoint_function
    return await dependant.call(**values)
  File "C:\Users\Tiago Barreto\Desktop\colab2mlops-main\.\source\api\main.py", line 77, in get_inference
    return "low income <=50K" if predict[0] <= 0.5 else "high income >50K"
